templates/compose/pterodactyl-with-wings.yaml

```
@@ -21,26 +22,27 @@ services:
       - MYSQL_USER=$SERVICE_USER_MYSQL
       - MYSQL_PASSWORD=$SERVICE_PASSWORD_MYSQL
     volumes:
-      - pterodactyl-db:/var/lib/mysql
-
+      - 'pterodactyl-db:/var/lib/mysql'
   redis:
-    image: redis:alpine
+    image: 'redis:alpine'
     healthcheck:
-      test: ["CMD-SHELL", "redis-cli ping || exit 1"]
+      test:
+        - CMD-SHELL
+        - 'redis-cli ping || exit 1'
       interval: 10s
       timeout: 1s
       retries: 3
-
   pterodactyl:
-    image: ghcr.io/pterodactyl/panel:latest
+    image: 'ghcr.io/pterodactyl/panel:latest'
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

---

[v1.11.11](https://github.com/orgs/pterodactyl/packages/container/panel/441605424?tag=v1.11.11)

```suggestion
    image: 'ghcr.io/pterodactyl/panel:v1.11.11'
```